### ISSUE TITLE
always set default language when deploy to server

### ISSUE THREAD

**Issue by catsoft** [yoolab/iron-router-i18n#44]
Something wrong but I could not solve it.Everything works fine as expected in my development machine but when deploy it to server "autoConfLanguage" not working. Am I missing something..

**Comment by martinopic** [comment]
have you tried to see if it's a client side or server side problem? try to disable autoConfLanguage server or client side on development and server (should work on both)

**Comment by catsoft** [comment]
Sorry but I couldn't understand should I have 
Router.configure({
    i18n: {
     .....
   }

both server side and client side. I have one config and nothing changes if I add or remove serverSide:true

**Comment by catsoft** [comment]
you can see it http://justrentacar.net.it turns "en" whatever browser language is.here is my full config
Router.configure({
    i18n: {
        serverSide: true,
        languages: ["en", "tr"],
        autoConfLanguage: true,
        exclude: {
            sitemap: '^\/sitemap\.xml',
        },
        setLanguage: function(lang) {
            this._language = lang;
            TAPi18n.setLanguage(lang);
        }
    }
});

**Comment by martinopic** [comment]
my suggestion was to try on server (en locally) this configuration: 

```
Router.configure({
i18n: {
  ...
  client: {
    autoConfLanguage: false
  },

  server:  {
    autoConfLanguage: false
 }

});

```
Disabling/enabling "autoConfLanguage" (setting it to true/false) on client and server side configuration and seeing which one is not working: client side implementation relies on ```navigator.language``` property (I'm giving for granted you are using exactly the same client/browser for testing so it should just work) while server side implementation is using NPM locale package which I think uses the HTTP accept-language header (here there could be a problem with server side package and/or something done by the server on http headers).

**Comment by catsoft** [comment]
I tried all combinations in locally and remote. On locally only client side implementation works but on remote non of them. And there is no error anywhere. So what is your suggestion.

**Comment by martinopic** [comment]
It seems Chrome is setting ```navigator.language``` in a weird way (https://alicoding.com/detect-browser-language-preference-in-firefox-and-chrome-using-javascript/), are you using Chrome? This patch should manage this case.

But I've tried to test your website and the configuration seems ok and it doesn't work with Firefox, unfortunately I can't easily debug it because the code is minified, it would be really useful if I could access a non minified version and/or have some way to replicate the issue.

**Comment by catsoft** [comment]
Yes I have tried before the code you use in the last commit for dedecting browser language.I think it would solve the problem. And about non minified version I dont know how can I do it. Can you describe me how to do.

**Comment by martinopic** [comment]
Second thought: I think this is a duplicate of #42 it's fixed in master branch and will be soon in 0.5.4 release (use master branch if you need to have it now). I will not mark it as duplicate because it made me discover this strange behaviour of Chrome ;-)
